build/build-assets.ps1

```diff
@@ -34,12 +34,15 @@ $pluginFolder = "${targetFolder}\App_Plugins\Contentment\";
 if (!(Test-Path -Path $pluginFolder)) {New-Item -Path $pluginFolder -Type Directory;}
 Copy-Item -Path "${ProjectDir}Web\UI\App_Plugins\Contentment\*" -Force -Recurse -Destination "${pluginFolder}";
 
+
 # HTML (Property Editors) - Copy and Minify (or just remove comments)
+$editorFolder = "${pluginFolder}\editors"
+if (!(Test-Path -Path $editorFolder)) {New-Item -Path $editorFolder -Type Directory;}
```

Comment:
I think I'm going to take a different approach to this.

The issue is that the "/App_Plugins/Contentment/editors" folder is missing is because there are no files in there. The simplest solution (for me, and less code), is to put a placeholder file in there.  I'm considering moving the "[_empty.html](https://github.com/leekelleher/umbraco-contentment/blob/master/src/Umbraco.Community.Contentment/DataEditors/_/_empty.html)" editor file (which is used for Data List).

Reply:
Sure. I guess you could just copy an empty file in there as well. 
Problem was just that "Set-Content" wouldn't automatically create the missing folder, unlike the Copy-Item command, which you use for the other files and folders. :-) 
I'll revert build-assets.ps1